backend/btrixcloud/pages.py

```diff
@@ -501,7 +501,11 @@ async def list_pages(
 
         return [PageOut.from_dict(data) for data in items], total
 
-    async def get_crawl_file_count(self, crawl_id: str):
+    async def get_crawl_page_count(self, crawl_id: str) -> int:
+        """Return total number of pages in db for crawl"""
+        return await self.pages.count_documents({"crawl_id": crawl_id})
```

Comment:
This works, but should also already be available via the `CrawlStats.done` field in the crawl model, which should be equal to this. Any discrepancy (hopefully there shouldn't be) should be counted as an error, since this is the number that we display to users of how many pages were crawled / and total pages for QA run.

Reply:
Ah never mind, I see that this is being compared against to determine when all pages were added!

Reply:
Yup! Just wanted to make sure we didn't cache incomplete numbers in the database :)